src/env.zig

```diff
@@ -61,6 +61,25 @@ pub const OperatingSystem = enum {
     // wAsM is nOt aN oPeRaTiNg SyStEm
     wasm,
 
+    pub const names = @import("root").bun.ComptimeStringMap(
+        OperatingSystem,
+        &.{
+            .{ "windows", OperatingSystem.windows },
+            .{ "win32", OperatingSystem.windows },
+            .{ "win64", OperatingSystem.windows },
+            .{ "win_x64", OperatingSystem.windows },
+            .{ "darwin", OperatingSystem.mac },
+            .{ "macos", OperatingSystem.mac },
+            .{ "macOS", OperatingSystem.mac },
+            .{ "mac", OperatingSystem.mac },
+            .{ "apple", OperatingSystem.mac },
+            .{ "linux", OperatingSystem.linux },
+            .{ "linux-gnu", OperatingSystem.linux },
+            .{ "gnu/linux", OperatingSystem.linux },
+            .{ "wasm", OperatingSystem.wasm },
```

Comment:
im surprised `win` is not in this list

and `win_x64` is really suspicious here. when arm is allowed this will be very confusing to parse.